iam_units/test_all.py

```diff
@@ -72,3 +79,41 @@ def test_units_emissions(context, value):
         # assert that conversion to CO2-equivalent works
         assert registry[f.format('ch4')].to(f.format('co2e'),
                                             context).magnitude == value
+
+
+def test_emissions_internal():
+    # Dummy units can be created
+    registry('0.5 _gwp').dimensionality == {'[_GWP]': 1.0}
+
+    # Context can be enabled
+    with registry.context('AR5GWP100', __a=1.0) as r:
+        # Context-specific values are activated
+        assert r('a_CH4 * a_N2O').to_base_units().magnitude == 28 * 265
+
+        # Context-specific conversion rules are available
+        r('0.5 t').to('_gwp')
+
+
+@pytest.mark.parametrize('units', ['t {}', 'Mt {}', 'Mt {} / yr'])
+@pytest.mark.parametrize('metric, expected_value', EMI_DATA)
+def test_convert_gwp(units, metric, expected_value):
+    # Bare masses can be converted
+
+    qty = registry.Quantity(1.0, units.format(''))
+    expected = registry(f'{expected_value} {units}')
+    assert convert_gwp(metric, qty, 'CH4', 'CO2') == expected
```

Comment:
can this be changed such that just a value is also accepted?

Reply:
You mean, just a magnitude? What would its units be?

Reply:
Well, in my thinking they would be part of the `*species` args, but I realize now that this would be inconsistent with your goal to get the species out of the units entirely...